docs/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -99,14 +102,20 @@ Additionally, even when the SDK is downloaded and initialized correctly, Sentry
 Furthermore, some browsers, like [Brave](https://brave.com/), have built-in ad-blockers that may block requests sent to our endpoint. Even if users deactivate your domain from blocking, Brave might continue to block [requests made from service workers](https://github.com/getsentry/sentry/issues/47912#issuecomment-1573714875).
 
 You can work around our CDN bundles being blocked by [using our NPM packages](#using-the-npm-package) and bundling our SDK with your app. However, the endpoint blockage can only be resolved by using a tunnel as explained below.
+
 </Expandable>
 
 <Expandable title="Using the `tunnel` Option">
 
 <PlatformSection supported={["javascript.nextjs"]}>
 
 <Note>
-  The Sentry Next.js SDK has a separate option to make setting up tunnels very straight-forward, allowing you to skip the setup below. See <PlatformLink to="/manual-setup/#configure-tunneling-to-avoid-ad-blockers">Configure Tunneling to avoid Ad-Blockers</PlatformLink> to learn how to set up tunneling on Next.js.
+  The Sentry Next.js SDK has a separate option to make setting up tunnels very
+  straight-forward, allowing you to skip the setup below. See{" "}
+  <PlatformLink to="/manual-setup/#configure-tunneling-to-avoid-ad-blockers">
+    Configure Tunneling to avoid Ad-Blockers
+  </PlatformLink>{" "}
```

Comment:
```suggestion
  straight-forward, allowing you to skip the setup below. See
  <PlatformLink to="/manual-setup/#configure-tunneling-to-avoid-ad-blockers">
    Configure Tunneling to avoid Ad-Blockers
  </PlatformLink>
```
I think the formatting looks weird with the extra line breaks, can we remove them?

Reply:
Yes! It must have been an autoformatter in my code editor that did this, let's remove.